--- a/project/mt7682_hdk/apps/mqtt_client/src/sys_init.c
+++ b/project/mt7682_hdk/apps/mqtt_client/src/sys_init.c
@@ -49,6 +49,7 @@
 
 #include "memory_map.h"
 #include "memory_attribute.h"
+#include "nvdm_boot.h"
 
 
 extern uint32_t chip_hardware_code(void);
@@ -324,14 +325,21 @@ void system_init(void)
 #ifdef MTK_WIFI_ROM_ENABLE
     move_iot_rom_data_to_ram();
 #endif
-    /* Connsys init will download FW, MT5932 need download N9 FW before any task created, to init ucHeap early */
-    connsys_init(NULL);
 
+#ifndef MTK_DEBUG_LEVEL_NONE
+    log_init(NULL, NULL, syslog_control_blocks);
+#endif
+
 #ifdef MTK_NVDM_ENABLE
+    nvdm_prepare_region();
     /* nvdm init */
     nvdm_init();
     user_check_default_value();
 #endif
 
+    /* Connsys init will download FW, MT5932 need download N9 FW before any task created, to init ucHeap early */
+    connsys_init(NULL);
+
 #ifndef MTK_DEBUG_LEVEL_NONE
 #ifdef MTK_NVDM_ENABLE
     log_init(syslog_config_save, syslog_config_load, syslog_control_blocks);
